ajax/libs/inferno-create-element/package.json

```diff
@@ -26,7 +26,7 @@
     {
       "basePath": "dist",
       "files": [
-        "**/inferno-create-element!(*.node)*"
+        "**/inferno-create-element!(*.node|*.es2015)*"
```

Comment:
Why this diff is in this commit?

Reply:
I didn't check if it's correct or not, sorry about that. The correct should be `"**/inferno-create-element!(*.node)*.+(js|map|css)"` used to exclude inferno-create-element.node.js and inferno-create-class.d.ts in the old versions. Please help me review again, thank you!